geometry_msgs/Point pos
geometry_msgs/Vector3 vel
geometry_msgs/Vector3 prefVelocity
geometry_msgs/Point[] goals 